client/components/d3chart/index.js

```diff
@@ -74,27 +70,27 @@ class D3Chart extends Component {
 		return [ ...props.data, ...orderedKeys ];
 	}
 
-	drawChart( node, params ) {
-		const { data, margin, type } = this.props;
+	drawChart( node ) {
+		const { data, height, margin, type, width } = this.props;
+		const params = this.getParams();
+		const adjParams = Object.assign( {}, params, {
+			height: height - margin.top - margin.bottom,
+			width: width - margin.left - margin.right,
```

Comment:
Lets move this width and height calculation to `getParams` function so the removal of the margins is done in one place.

Reply:
Should be fixed here: https://github.com/woocommerce/wc-admin/pull/961/commits/5a447c852f1a61d15c11fd941cdc24f4225c4239.